examples/basic-examples/modules/ces/ces.tf

```diff
@@ -5,16 +5,16 @@
 #}
 
 resource "opentelekomcloud_ces_alarmrule" "alarm_rule" {
-  alarm_name        = "${var.alarm_name}"
-  alarm_description = "${var.alarm_desc}"
+  alarm_name        = var.alarm_name
+  alarm_description = var.alarm_desc
   metric {
     #"namespace" = "SYS.ECS"
```

Comment:
useless comment?

Reply:
I was not removing comments, examples still to be checked